virtcontainers/container.go

```diff
@@ -633,6 +636,9 @@ func (c *Container) rollbackFailingContainerCreation() {
 	if err := c.removeDrive(); err != nil {
 		c.Logger().WithError(err).Error("rollback failed removeDrive()")
 	}
+	if err := c.sandbox.storage.deleteContainerResources(c.sandboxID, c.id, nil); err != nil {
+		c.Logger().WithError(err).Error("rollback failed storage deleteContainerResources()")
+	}
```

Comment:
The change here is related to the rollback of container creation, and it should be in its own commit. Please commit this separately (but in the same PR), and provide a nice commit message to explain what you did here.

Reply:
This change is put to commit "sandbox: rollback container resources when create fail"~